MAUI/Effects-View/Effects/Rotation.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
+description: Learn here all about rotation effect support in Syncfusion .NET MAUI Effects View (SfEffectsView) control and more.
+platform: maui
+control: Effects View
+documentation: ug
+---
+
+# Rotation Effect in .NET MAUI Effects View (SfEffectsView)
+
+[SfEffects.Rotation](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffects.html#Syncfusion_Maui_Core_SfEffects_Rotation) provides a circular movement to [SfEffectsView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html) around the center of the [SfEffectsView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html) based on the specified [Angle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfEffectsView.html#Syncfusion_Maui_Core_SfEffectsView_Angle).
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<syncEffectsView:SfEffectsView
+    TouchDownEffects="Rotation"
+    Angle="180">
+</syncEffectsView:SfEffectsView>
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var effectsView = new SfEffectsView
+{
+    TouchDownEffects = SfEffects.Rotation,
+    Angle = 180,
+};
+
+{% endhighlight %}
+
+{% endtabs %}
```

Comment:
Add the gif or image for rotation animation

Reply:
Added the GIF for rotation animation.